app/controllers/__init__.py

```diff
@@ -32,5 +32,5 @@
                                ProjectDatabaseRetrievePasswordView, ProjectDatabaseAdminRetrievePasswordView, DatabaseStatsView)
 from .billing_invoice import (
     BillingInvoiceView, BillingInvoiceNotificationView)
-from .system_status import SystemStatusView, SystemSummaryView
```

Comment:
Hello @mirembe-mariam. Please only delete the deleted class resource `SystemStatusView` from the statement not the whole import statement. This will make `SystemSummaryView` unavailable for use .
It would be something like `from .system_status import SystemSummaryView`